Lib/glyphsLib/builder/axes.py

```diff
@@ -165,40 +192,46 @@ def to_designspace_axes(self):
                 continue
         # See https://github.com/googlefonts/glyphsLib/issues/280
         elif font_uses_axis_locations(self.font):
-            # Build the mapping from the "Axis Location" of the masters
+            # If all masters have an "Axis Location" custom parameter, only the values
+            # from this parameter will be used to build the mapping of the masters and
+            # instances
             # TODO: (jany) use Virtual Masters as well?
+            #       (jenskutilek) virtual masters can't have an Axis Location parameter.
             mapping = {}
             for master in self.font.masters:
                 designLoc = axis_def.get_design_loc(master)
                 userLoc = axis_def.get_user_loc(master)
+                if userLoc is None:
+                    # May happen if the custom parameter is disabled
```

Comment:
the method `get_user_loc` can never return None, see how at line 367 the variable is initialized to something that is not None:

user_loc = self.default_user_loc

Even if the master custom parameter is missing or disabled, it will still return something. Please remove this `if userLoc is None:` bloc